pkg/ruler/rules/objectclient/rule_store.go

```diff
@@ -0,0 +1,164 @@
+package objectclient
```

Comment:
Why extra package?

Reply:
This isn't consistent with the other store in this package. However, this is consistent with how the `pkg/alertmanager/alerts` package has it's storage clients. It's also similar to `pkg/chunks/` does. In a future PR I will nest the configdb rulestore client into a separate package as well.